### ISSUE TITLE
Is there an actively maintained fork of this somewhere?

### ISSUE THREAD

**Issue by chmac** [ushahidi/SMSSync#491]
I had a look on fdroid / play store, but I couldn't find anybody publishing an active fork of this.

There are 440 forks here on GitHub though. Anybody know if one of them is actively maintained?

**Comment by 091530777777** [comment]
> I had a look on fdroid / play store, but I couldn't find anybody publishing an active fork of this.
> 
> There are 440 forks here on GitHub though. Anybody know if one of them is actively maintained?

**Comment by chweyakevin** [comment]
Nope. I have taken some time to go through some of them but none seems to be maintained. Maybe most people moved to the cloud option which is cheaper and convenient nowadays.

**Comment by Cookie403** [comment]
@YveOms Can you let me know what good cloud alternatives are to SMSSync ?

**Comment by AchoArnold** [comment]
Hello @Cookie403 / @chmac 
You can check out this open source cloud alternative https://httpsms.com which I'm currently working on.

**Comment by sicaboy** [comment]
> Hello @Cookie403 / @chmac You can check out this open source cloud alternative https://httpsms.com which I'm currently working on.

this is only free for the first 200 messages. it wouldn't be a good alternative

**Comment by AchoArnold** [comment]
@sicaboy 
You can always self-host if you don't want to pay for the cloud version, servers cost money, unfortunately.

**Comment by sicaboy** [comment]
Oops I misunderstood. Thanks for sharing. 

> @sicaboy 
> 
> You can always self-host if you don't want to pay for the cloud version, servers cost money, unfortunately.
